nemo_text_processing/text_normalization/hi/data/whitelist/alternatives_all_format.tsv

```diff
@@ -0,0 +1,14 @@
+st	street
```

Comment:
Since this is English, is it necessary for HI TN?

Reply:
I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?

Reply:
same suggestion as for the above file